cfme/tests/configure/test_access_control.py

```diff
@@ -1929,3 +1930,70 @@ def test_tenant_visibility_miq_ae_namespaces_all_parents():
         startsin: 5.5
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.ignore_stream('5.10')
+@test_requirements.rbac
+def test_tags_manual_features():
+    """
+    Test that user can edit tags of an VM when he has role created by disabling 'Everything'
+    and then enabling every other checkbox.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Configuration
+        caseimportance: high
+        initialEstimate: 1/5h
+        startsin: 5.11
+        setup:
+            1. Have an infra provider added and testvm created
+        testSteps:
+            1. Create new role. Disable 'Everything' and then manually enable each check box
+                except 'Everything'.
+            2. Create new group based on this role; Create new user as a member of this group
+            3. Login as newly created user
+            4. Navigate to testing vm and try to Policy -> Edit Tags
+        expectedResults:
+            1. New role created
+            2. New group created; New user created
+            3. Login successful
+            4. Edit Tags screen displayed; no error in evm.log
+    Bugzilla:
+        1684472
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.provider([OpenshiftProvider], override=True)
+@test_requirements.rbac
```

Comment:
missing `tier` as well here

Reply:
Note it defaults to component (1), if that is correct here we don't need an explicit marker.

Reply:
Tier added.